foreman-bootloaders-redhat/foreman-bootloaders-redhat.spec

```diff
@@ -0,0 +1,55 @@
+%define _binaries_in_noarch_packages_terminate_build 0
+
+Name: foreman-bootloaders-redhat
+Version: 201705231433
+Release: 1%{?dist}
+Summary: Binaries of Grub1, Grub2 and Shim bootloaders
+
+Group: Applications/System
+License: GPLv2+ and GPLv3+ and BSD
+URL: https://github.com/theforeman/foreman-bootloaders
+Source0: http://downloads.theforeman.org/foreman-bootloaders/%{name}-%{version}.tar.bz2
+BuildArch: noarch
+
+Requires: /var/lib/tftpboot
+
+%description
+The package contains binaries of Grub and Grub2 bootloaders for PXE/BOOTP use
+on TFTP Foreman Proxy extracted from Red Hat Enterprise Linux 6 and 7. This
+noarch package contains binaries for multiple architectures, including x86_64,
+i386, ARM64, PPC64 and PPC64 LE.
+
+
+%prep
+%setup -q -n tftpboot
+
+
+%build
+
+
+%install
+install -d -m0755 %{buildroot}%{_sharedstatedir}/tftpboot
+cp -p -r grub grub2 %{buildroot}%{_sharedstatedir}/tftpboot
+
+
+%files
+%{_sharedstatedir}/tftpboot/grub/grubx64.efi
+%{_sharedstatedir}/tftpboot/grub/grubia32.efi
+%{_sharedstatedir}/tftpboot/grub2/grubaa64.efi
+%{_sharedstatedir}/tftpboot/grub2/grubx64.efi
+%{_sharedstatedir}/tftpboot/grub2/shimaa64-redhat.efi
+%{_sharedstatedir}/tftpboot/grub2/shimia32.efi
+%{_sharedstatedir}/tftpboot/grub2/grubia32.efi
+%{_sharedstatedir}/tftpboot/grub2/grubppc64.elf
+%{_sharedstatedir}/tftpboot/grub2/shimx64-redhat.efi
+%{_sharedstatedir}/tftpboot/grub2/grubppc64le.elf
+%{_sharedstatedir}/tftpboot/grub2/shimaa64.efi
+%{_sharedstatedir}/tftpboot/grub2/shimia32-redhat.efi
+%{_sharedstatedir}/tftpboot/grub2/grubppc64.efi
```

Comment:
This is supposed to be symlink to *elf files (two of them) to workaround this problem: http://projects.theforeman.org/issues/16706 I will verify if these are really symlinks and fix this if not.